lib/aptible/cli/subcommands/backup.rb

```diff
@@ -0,0 +1,46 @@
+module Aptible
+  module CLI
+    module Subcommands
+      module Backup
+        def self.included(thor)
+          thor.class_eval do
+            include Helpers::Token
+            include Helpers::Database
+
+            desc 'backup:restore [--handle HANDLE] [--size SIZE_GB]',
+                 'Restore a backup'
+            option :handle
+            option :size, type: :numeric
+            define_method 'backup:restore' do |backup_id|
+              backup = Aptible::Api::Backup.find(backup_id, token: fetch_token)
+              fail Thor::Error, "Backup ##{backup_id} not found" if backup.nil?
+              handle = options[:handle]
+              handle ||= "#{backup.database.handle}-at-" \
+                         "#{backup.created_at.strftime '%Y-%m-%d-%H-%M'}"
+
+              opts = {
+                type: 'restore',
+                handle: handle,
+                disk_size: options[:size]
+              }.delete_if { |_, v| v.nil? }
+
+              operation = backup.create_operation!(opts)
+              say "Restoring backup into #{handle}"
+              attach_to_operation_logs(operation)
+            end
+
+            desc 'backup:list DB_HANDLE', 'List backups for a database'
+            option :environment
+            define_method 'backup:list' do |handle|
+              # TODO: Expose pagination from aptible-resource
```

Comment:
What use case do you have in mind? If we're fetching all of them, this doesn't seem super useful and might confuse users (that expect the command to work faster with a limit) rather than serve them.

One use case I can think of is to make it easier to not break backwards compatibility if / when we support paging in aptible-resource, but in this case I'd rather and support this now.